### ISSUE TITLE
Can't achieve high accuracy after training from scratch

### ISSUE THREAD

**Issue by Zephyr06** [facebook/C3D#389]
Hello, I recently used pytorch to train C3D model from scratch, which is the basis for my another work. But I found I can't achieve high accuracy after many epochs(I tried my different lr, and the epochs are more than 400) . Actually, the final accuracy is just the probability of random selection from my label set. 

Another problem is that I found the final accuracy for action recognition just keeps the same after simply 20 epochs or so, though the training loss keeps changing after each epoch. So could anybody help me with this two problems? 

Thanks a lot!

**Comment by dutran** [comment]
This can be anything, e.g. bug in your training script, bug in some layers. I guess you have to debug layer by layer to see what cause your training stuck.
